### PR TITLE
V 1.0

### PR DESCRIPTION
**Pull request by youssef-bounit**
Version 1.0

### CODE REVIEW

File: advanceddashboard/lib/Controller/PageController.php [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+		$sqlInternalShares->execute();
+		$totalInternalShares = $sqlInternalShares->fetchColumn();
+
+		$this->sortByKey($topUsersByFilesCount, 'filecount', true);
+
+		$this->sortByKey($topUsersByUsedSpace, 'usedSpace', true);
+
+		// echo "<pre>";
+		// var_dump($totalSahredFiles);
+		// echo "</pre>";
```
Comment (bondif) [inline advanceddashboard/lib/Controller/PageController.php]: Can you remove this, please?

File: advanceddashboard/lib/Controller/PageController.php [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+
+use OCP\AppFramework\Http\Request;
+use OCP\AppFramework\Http\RedirectResponse;
+
+
+class PageController extends Controller {
+	private $userId;
+
+	private $config;
+    private $userManager;
```
Comment (bondif) [inline advanceddashboard/lib/Controller/PageController.php]: Indent this, please.

File: advanceddashboard/.travis.yml
Comment (bondif) [inline advanceddashboard/.travis.yml:1]: This file is not needed, since we're not using TravisCI

File: advanceddashboard/README.md
Comment (bondif) [inline advanceddashboard/README.md:1]: Remove this, since we already have one.

File: advanceddashboard/appinfo/info.xml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<info xmlns:xsi= "http://www.w3.org/2001/XMLSchema-instance"
+      xsi:noNamespaceSchemaLocation="https://apps.nextcloud.com/schema/apps/info.xsd">
+    <id>advanceddashboard</id>
+    <name>Advanced Dashboard</name>
+    <summary>Advanced Dashbord</summary>
+    <description><![CDATA[a Simple Advanced Dashboard]]></description>
```
Comment (bondif) [inline advanceddashboard/appinfo/info.xml]: "Advanced Statistics Dashboard"

File: advanceddashboard/appinfo/info.xml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    <version>1.0</version>
+    <licence>agpl</licence>
+    <author mail="youssefbounit768@gmail.com" homepage="https://drivetest.cnrst.ma">Youssef Bounit</author>
```
Comment (bondif) [inline advanceddashboard/appinfo/info.xml]: Remove or change the `homepage` link

File: advanceddashboard/appinfo/info.xml [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    <namespace>AdvancedDashboard</namespace>
+    <category>dashboard</category>
+    <bugs>https://youssef-example.org</bugs>
```
Comment (bondif) [inline advanceddashboard/appinfo/info.xml]: bugs link: https://github.com/bondif/nextcloud-advanced-dashboard/issues

File: advanceddashboard/appinfo/info.xml
Comment (bondif) [inline advanceddashboard/appinfo/info.xml:1]: Please add a website tag pointing to https://github.com/bondif/nextcloud-advanced-dashboard

File: advanceddashboard/appinfo/routes.php
Comment (bondif) [inline advanceddashboard/appinfo/routes.php:1]: Fix indentation

File: advanceddashboard/settings/admin.php [hunk 6]
```diff
@@ -0,0 +1,34 @@
+<?php
+namespace OCA\YourAppName\Settings;
```
Comment (bondif) [inline advanceddashboard/settings/admin.php]: Fix namespace

File: advanceddashboard/templates/settings/index.php
Comment (bondif) [inline advanceddashboard/templates/settings/index.php:1]: Remove the comments

**Review by bondif — COMMENTED** [review]
Remove the parent directory, please.

**Comment by youssef-bounit** [comment]
I think everything is good now

**Review by bondif — APPROVED** [review]
Thanks

**Pull request merged**